static/css/main.css

```diff
@@ -11,7 +11,7 @@ label { display:inline-block; font-weight: 600; margin-bottom: 0.25rem; width:20
     position: relative;
 }
 
-.btn--hidden, .supplementary-data--hidden {
+.btn--hidden, .supplementary-data--hidden, .cir-metadata--hidden {
```

Comment:
Wouldn't it make more sense to call it `.hidden` class? Instead of creating 3? Or even doing inline style: "style=display:none"?

Reply:
Problem with using `.hidden` is that I then can't differentiate between them, so I can't hide just one of the dropdowns, I do need it separate. That wouldn't be an issue with inline styling but my preference is to stick with the css class as it separates the styling and logic, and I don't then need to manually do something like `display=block` or similar when undoing the hide

Reply:
Ok, makes sense.